src/base-components/Listing/BaseListing.js

```diff
@@ -268,16 +276,23 @@ export default {
             axios.get(this.url, options).then(response => this.populateCurrentStateAndData(response.data.data), error => {
                 // TODO handle error
             });
+
+            if(updateUrl) {
+                this.updateUrl(options.params);
+            }
         },
 
-        filter(column, value) {
+        //FIXME: filter can be called by child listing components on create to set default filters
+        filter(column, value, loadData = true) {
             if (value == '') {
                 delete this.filters[column];
             } else {
                 this.filters[column] = value;
             }
             // when we change filter, we must reset pagination, because the total items count may has changed
-            this.loadData(true);
+            if(loadData) {
```

Comment:
nerozumiem, kedy chces volat filter s loadData=false?

Reply:
v setParamsFromUrl, nastavujes jednotlive filtre z urlky. Ale nechces pri kazdom nastaveni filtru loadovat data, chces loadovat data len na konci naraz.

Reply:
aha nieco ako, ze si najprv nastavit jeden filter, potom druhy a na konci das Apply filters a az vtedy to loadne? take nieco?

Reply:
jo bolo to tak, ale pozeram ze som to potom zmenil a toto je uz neaktualne :)